analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -348,11 +348,17 @@ data class GemSpec(
                 RemoteArtifact.EMPTY
             }
 
+            val authors = json["authors"]?.textValue()
+                ?.split(',')
+                ?.map {
+                    it.trim()
+                }?.toSortedSet() ?: sortedSetOf()
```

Comment:
How about making use of the `Packagemanager.parseAuthorString` helper function here?

Reply:
Not a bad idea, but why is the split method limited to [two substrings](https://github.com/oss-review-toolkit/ort/blob/4d14e50170ca536b5f47379291b8eb54de980112/analyzer/src/main/kotlin/PackageManager.kt#L294)? That makes it not quite feasible for multiple authors.

Reply:
Sorry, my bad, the purpose of `parseAuthorString` (note the singular "author" in the name) is to split e.g. an email address from the author name, for a single author. So it doesn't really help here.